src/components/galleryCard/GalleryCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Card, CardTitle, CardImage, CardContent, CardFooter } from '../card'
+import { Tag } from '../../components/tag'
+import { H5, H6, P } from '../typography'
+import { colors } from '../../theme/colors'
+import { GalleryCardProps } from './GalleryCard.types'
+
+const StyledTag = styled(Tag)`
+  display: flex;
+  transform: translate(5px, -35px);
+  color: indigo;
+  padding: 6px;
```

Comment:
I'd remove this `padding` override until design see it - it should be the same as the `Tag` component for consistency.